AirportTicketBookingSystem/Domain/FlightManagement/Flight.cs

```diff
@@ -5,20 +5,23 @@
 
 namespace AirportTicketBookingSystem.Domain.FlightManagement
 {
-    public  class Flight
+    public record Flight
     {
-       [Required(ErrorMessage ="flight id must be entered.")]
-        
-       public  int FlightId { get; init; }
+        [Required(ErrorMessage = "flight id must be entered.")]
+
+        public int FlightId { get; init; }
         public decimal Price { get; set; }

```

Comment:
why set not init ?

Reply:
the price of the unbooked tickets might change in future as the time of the departure gets close.